src/utils/configuration.ts

```diff
@@ -0,0 +1,28 @@
+let yaml = require('js-yaml')
+let fs = require('fs')
+let path = require('path')
+
+export interface SnapshotConfiguration {
+  widths?: [number],
+  'min-height'?: number,
+  'enable-javascript'?: boolean
```

Comment:
I thought we decided not to include this option to reduce confusion?

Reply:
Gone.